src/components/cards/card.css

```diff
@@ -13,7 +14,7 @@
     background: $ui-white;
     border: 1px solid $ui-tertiary;
     width: 10px;
-    z-index: 99;
+    z-index: 10;
```

Comment:
Shouldn't these be converted to variables as well?

Reply:
I don't think it's necessary. The card component creates a stacking context, so the z-index values of left-card and right-card specify only the relative order of elements within the card and are thus independent from the rest of the app.